org.scala-ide.sdt.debug/src/org/scalaide/debug/internal/model/ScalaThread.scala

```diff
@@ -62,7 +65,8 @@ abstract class ScalaThread private(target: ScalaDebugTarget, val threadRef: Thre
 
   // Members declared in org.eclipse.debug.core.model.ISuspendResume
 
-  override def canResume: Boolean = suspended // TODO: need real logic
+  override def canResume: Boolean = suspended && // TODO: needs real logic
```

Comment:
I don't know what is the real logic which an author had in  mind. I only added one condition. Thus I didn't remove TODOs.